ipam/azure.go

```diff
@@ -115,6 +118,13 @@ func (s *azureSource) refresh() error {
 		return err
 	}
 
+	log.Printf("[ipam] xml name received:%v interfaces:%d", doc.XMLName, len(doc.Interface))
+
+	if len(doc.Interface) == 0 {
```

Comment:
can we log doc in this case?

Reply:
if we log doc won't it log the entire struct with all the ip configs?